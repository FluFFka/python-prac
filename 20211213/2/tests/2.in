[-508, 445, 413, 119, -444, -828, -748, 121, -32, -212, -85, -496, 634, -154, -520, -626, 324, 513, 130, -858, -104, -986, 418, 829, -494, 165, 674, 489, 163, -88, -620, -552, 976, 944, 447, 131, -300, -943, 458, -66, -459, -68, 479, -770, 134, -442, 80, 801, -885, 56, 443, 630, -715, 738, 60, 656, -578, -773, 971, -7, 622, 600, -519, -467, -466]
