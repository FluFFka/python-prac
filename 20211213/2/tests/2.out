[-986, -943, -885, -858, -828, -773, -770, -748, -715, -626, -620, -578, -552, -520, -519, -508, -496, -494, -467, -466, -459, -444, -442, -300, -212, -154, -104, -88, -85, -68, -66, -32, -7, 56, 60, 80, 119, 121, 130, 131, 134, 163, 165, 324, 413, 418, 443, 445, 447, 458, 479, 489, 513, 600, 622, 630, 634, 656, 674, 738, 801, 829, 944, 971, 976]
